### ISSUE TITLE
Error al conectar con el servidor

### ISSUE THREAD

**Issue by asyba** [NanMetal/CATracking#1]
Puede ser que ya no ande? me tira eso cuando intento buscar un numero que existe y lo puedo ver por https://www.correoargentino.com.ar/formularios/oidn

_Error al conectar con el servidor_

**Comment by NanMetal** [comment]
Sí, puede ser, ya que la respuesta es KO (knockout). Es probable que hayan cerrado o bloqueado algo en sus servidores. Recién probé y tampoco funciona la app de android
